eth-connector-tests/Cargo.toml

```diff
@@ -28,8 +28,8 @@ hex = "0.4.3"
 ethabi = "18.0"
 rlp = { version = "0.5.0", default-features = false }
 aurora-engine-migration-tool = { git = "https://github.com/aurora-is-near/aurora-engine-migration-tool.git", tag = "0.2.2" }
-aurora-workspace-eth-connector = { git = "https://github.com/aurora-is-near/aurora-workspace.git", tag = "0.4.1" }
-aurora-workspace-utils = { git = "https://github.com/aurora-is-near/aurora-workspace.git", tag = "0.4.1" }
+aurora-workspace-eth-connector = { git = "https://github.com/olga24912/aurora-workspace", branch = "add-near-plugin" }
+aurora-workspace-utils = { git = "https://github.com/olga24912/aurora-workspace", branch = "add-near-plugin" }
```

Comment:
Just so you don't forget to update it before the merge.

Reply:
fixed: https://github.com/aurora-is-near/aurora-eth-connector/pull/71/commits/892c5aa92502184b485c4c591d711b5b4f5f5c80